onesignal-SDK.py

```diff
@@ -0,0 +1,28 @@
+from onesignal_sdk.client import Client
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+app_id = os.environ.get("ONESIGNAL_APP_ID")
+api_key = os.environ.get("API_KEY")
+
+if not app_id or not api_key:
+    raise ValueError("ONESIGNAL_APP_ID or API_KEY is missing from the environment variables.")
+
+client = Client(app_id=app_id, rest_api_key=api_key)
+
+def send_notification():
+    notification_body = {
+        'contents': {
+          "fr": "Votre ordonnance du patient X arrive à expiration. Veuillez la renouveler.",
+          "en": "Your prescription for patient X is about to expire. Please renew it."
```

Comment:
Even though we would make the message dynamic in the future, we could make it generic now until we connect to the DB.
Maybe something like:
`A prescription is about to expire open the app to review.`

Reply:
I've changed

Reply:
I still see the old version